### ISSUE TITLE
New artwork for house/person scene

### ISSUE THREAD

**Issue by amanda-phet** [phetsims/number-line-distance#31]
(1) I don't know how this happened, but I gave you the wrong image asset for the person. I will have to update ai files, but for now here is the correct PNG for the person. 

(2) I have also created a background image for the scene. Ideally it is placed so that the house step lines up with the edge of the green grass (see this in the mockup below). I've also included the fire hydrant, but if that's problematic I can remove it. I just like this size a bit better, and I put a plant in front of it. For the sake of time, I'm going to include a few options below, since you might be working before I am able to respond to questions. There might be issues with the width or height, so just let me know where I need to make tweaks. 

# Mockup:
![image](https://user-images.githubusercontent.com/7852916/123360567-c4153900-d52a-11eb-8f15-fdf58488c370.png)

# Artwork:
(1) Person: 
![person](https://user-images.githubusercontent.com/7852916/123361163-cb891200-d52b-11eb-9f36-66e6acaa759f.png)

(2a) Simple background: 
![grass-sidewalk](https://user-images.githubusercontent.com/7852916/123361206-e065a580-d52b-11eb-8787-e0883e0f77b0.png)

(2b) Detailed background: 
![grass-sidewalk-detail](https://user-images.githubusercontent.com/7852916/123361291-0723dc00-d52c-11eb-833a-e4e8043db0a1.png)

(2c) Rectangular background with sidewalk lines: @jbphet mentioned that it might work to take this rectangular image and transform it to have perspective. For reference, I am using a 55º (and 125º) angle in my artwork, so if you can match that angle that would be great. If you aren't sure how to do this, check with @jbphet . We thought it might be nice to have sidewalk lines, but we weren't sure how it would look in practice, and I couldn't quickly mock it up. There are two pieces: one to distort, which is green and gray, and one piece that is the bottom that shouldn't have any distortion. 
![grass-sidewalk-distort](https://user-images.githubusercontent.com/7852916/123361405-3b979800-d52c-11eb-9ab7-8b44e998793b.png)

![sidewalk-edge](https://user-images.githubusercontent.com/7852916/123361418-42bea600-d52c-11eb-875e-b14dca820795.png)

**Comment by amanda-phet** [comment]
We also adjusted the perspective on the house, and I need to give you that new image.

**Comment by amanda-phet** [comment]
It's looking great @SaurabhTotey . 

When you changed the artwork for the person, she got smaller. Is it possible to resize her so she's more like the size in previous versions? If it's due to the artwork I exported, let me know how I should change the size. 

Previous size: 
![image](https://user-images.githubusercontent.com/7852916/123835435-c3332d00-d8c5-11eb-90d1-f7b031c17fa4.png)


Current: 
![image](https://user-images.githubusercontent.com/7852916/123835396-b3b3e400-d8c5-11eb-9f90-62ec639ae61e.png)

**Comment by SaurabhTotey** [comment]
It seems like the new person artwork is indeed smaller than the previous person artwork, but I can scale up the image within the code. I have scaled the person up so that their head now nearly reaches the top of the door. Is this size good, and if so, should I make a new dev version with this change?

**Comment by amanda-phet** [comment]
Looks great, thanks. We can wait on a new dev version until you have any other changes that you'd like reflected in the next version.
